src/configs/common/v1-base-config

```diff
@@ -16,3 +16,5 @@ opt_enable SHOW_CUSTOM_BOOTSCREEN
 echo "# ${CONFIG_NAME} - ${MARLIN_VERSION} - ${V1_VERSION}" > README.md
 cat $CFGDIR/common/Readme_header.md >> README.md
 
+opt_enable \
+    EEPROM_AUTO_INIT
```

Comment:
Isn't this also enabled at https://github.com/V1EngineeringInc/MarlinBuilder/blob/main/src/configs/common/cnc-config#L35 ?

Is the intent here to enable it for 3DP and ZenXY as well?

I don't think it resets config on every flash though. AFAIK it will only trigger if EEPROM was empty or if the format has changed sufficiently for Marlin to be unable to read its contents.

Reply:
Yup, I think this is best for all builds.

I ran into this while tuning acceleration and jerk. I had to keep resetting the eeprom manually after a flash.